docs/UserGuide.md

```diff
@@ -122,6 +122,23 @@ EduContacts is a **desktop app for educators in tertiary institutions to manage
 
 6. Refer to the [Features](#features) below for details of each command.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## Summary of a `Person`
+
+This table will explain the fields that a `Person` in EduContacts possesses and its respective constraints. Each person
+in EduContacts is assumed to be Singaporean. All fields should not be blank upon the initialisation of a Student, except for `Module`.
```

Comment:
```suggestion
in EduContacts is assumed to be Singaporean. You should not leave any fields blank when adding a person to EduContacts, except for `Module`.
```

I think we can make this line more user-centric as such. I also think we should change student to person since this is a summary of a `Person`. Using student in this context could confuse users.